format_parser.gemspec

```diff
@@ -36,6 +36,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'faraday', '~> 0.13'
   spec.add_dependency 'faraday_middleware', '~> 0.14'
   spec.add_dependency 'measurometer', '~> 1'
+  spec.add_dependency 'nokogiri', '~> 1'
```

Comment:
Oh noes, please don't add nokogiri 🙈  Do we _actually_ need it?

Reply:
That comes back to my question about what we want to do about XMP 🤷🏻 If we want XMP to be parsed into accessible object as we do with Exif, then I think we'll need something like nokogiri. We could store the XMP as a string though and defer the responsibility of parsing it to the caller?